articles/virtual-machines/extensions/diagnostics-linux.md

```diff
@@ -559,23 +562,36 @@ BytesPerSecond | Number of bytes read or written per second
 
 Aggregated values across all disks can be obtained by setting `"condition": "IsAggregate=True"`. To get information for a specific device (for example, /dev/sdf1), set `"condition": "Name=\\"/dev/sdf1\\""`.
 
-## Installing and configuring LAD 3.0 via CLI
+## Installing and configuring LAD 3.0
+
+### Azure CLI
 
-Assuming your protected settings are in the file PrivateConfig.json and your public configuration information is in PublicConfig.json, run this command:
+Assuming your protected settings are in the file ProtectedSettings.json and your public configuration information is in PublicSettings.json, run this command:
 
 ```azurecli
-az vm extension set *resource_group_name* *vm_name* LinuxDiagnostic Microsoft.Azure.Diagnostics '3.*' --private-config-path PrivateConfig.json --public-config-path PublicConfig.json
+az vm extension set --publisher Microsoft.Azure.Diagnostics --name LinuxDiagnostic --version 3.0 --resource-group <resource_group_name> --vm-name <vm_name> --protected-settings ProtectedSettings.json --settings PublicSettings.json
```

Comment:
would this work with 3.0.125 or 128 etc? Shouldn't we put a * at the end?

Reply:
it will just pick the most recent version. i've done this with OMS before with `1.13` and it pulls the most recent subversion without issue